### ISSUE TITLE
Add production setup to website

### ISSUE THREAD

**Issue by knownasilya** [miguelcobain/ember-leaflet#14]
The following is required for Leaflet to render assets correctly when doing a production build:

```js
var app = new EmberApp(defaults, {
    // Add options here
    fingerprint: {
      exclude: [
        'images/layers-2x.png',
        'images/layers.png',
        'images/marker-icon-2x.png',
        'images/marker-icon.png',
        'images/marker-shadow.png'
      ]
    }
  });
```

**Comment by miguelcobain** [comment]
Confirm. We need to add this note to the docs.

**Comment by miguelcobain** [comment]
Solved in #15 and https://github.com/miguelcobain/ember-leaflet/commit/334cb9bbafb651e575464cca3f9224018a3dc32a.
